src/module/cache-container/config/security/view/add-role-modal.html

```diff
@@ -13,8 +13,8 @@ <h4 class="modal-title" id-generator="add_role_modal.title" translate>PERMISSION
           <input id-generator="role.name" type="text" class="form-control" ng-model="ctrl.role.name" ng-readonly="!ctrl.isNewRole">
         </div>
         <div class="col-md-2 pull-left" style="padding-left: 0">
-          <i class="fa fa-info-circle ng-scope" tooltip="The security role name" tooltip-trigger="" tooltip-animation="true"
-             tooltip-placement="top"></i>
+          <i class="fa fa-info-circle ng-scope" tooltip="The security role name"
+             tooltip-placement="left-bottom"></i>
```

Comment:
Based upon a quick grep, this also applies to `src/components/cache-security/view/cache-security.html`